### DISCUSSION TITLE
pre-built project

### DISCUSSION THREAD

**Discussion by Sinornithosaurus** [Sixze/ALS-Refactored#67] [Q&A] [answered]
Hey all, would anybody have a pre-built project I could test out in UE5? 

I'm trying to compile and I keep getting an error, 'Could not be compiled. Try rebuilding from source manually'. Honestly I'm super new so I have no idea what I'm doing. If anyone's interested or knows a fix, I'm currently using a blueprint project with a C++ class, I'm not sure if that's the way to go.

**Comment by Sixze** [comment]
Generate and open the code project in the Visual Studio or Rider and try to compile it from there to see a list of errors.

**Reply by Sinornithosaurus** [reply]
Project - Refactored (I called it this) 

I got these two errors:
File - UnrealBuildTool, Line 1
Error - Could not find definition for module 'AnimationBlueprintLibrary', (referenced via Target -> ALSEditor.Build.cs)
&
File - Microsoft.MakeFile.Targets, Line 45
Error - MSB3073	The command ""C:\Program Files\Epic Games\UE_5.0EA\Engine\Build\BatchFiles\Build.bat" RefactoredEditor Win64 Development -Project="C:\Users\callu\Documents\Unreal Projects\Refactored\Refactored.uproject" -WaitMutex -FromMsBuild" exited with code 6.

**Reply by Sinornithosaurus** [reply]
The output was as follows:

Build started...
1>------ Build started: Project: Refactored, Configuration: Development_Editor x64 ------
1>'"C:\Program Files\Epic Games\UE_5.0EA\Engine\Build\BatchFiles\GetDotnetPath.bat"' is not recognized as an internal or external command,
1>operable program or batch file.
1>Win64 using Manual SDK 10.0.16299.0
1>Invalidating makefile for RefactoredEditor (ALS.uplugin has been added)
1>UnrealBuildTool : error : Could not find definition for module 'AnimationBlueprintLibrary', (referenced via Target -> ALSEditor.Build.cs)
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.MakeFile.Targets(45,5): error MSB3073: The command ""C:\Program Files\Epic Games\UE_5.0EA\Engine\Build\BatchFiles\Build.bat" RefactoredEditor Win64 Development -Project="C:\Users\callu\Documents\Unreal Projects\Refactored\Refactored.uproject" -WaitMutex -FromMsBuild" exited with code 6.
1>Done building project "Refactored.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========

**Comment by Sixze** [comment, marked as answer]
UE 5.0 Early Access is no longer supported. Use UE 5.0 Preview.

**Reply by Sinornithosaurus** [reply]
Thanks, I can’t see the option for preview but I guess that’s because they rolled out the full release. I’ll keep an eye on things 👀

**Reply by Sixze** [reply]
There are some new compilation errors in the final version of the engine, but I'll fix them soon.

**Reply by teixeiraJonas** [reply]
> There are some new compilation errors in the final version of the engine, but I'll fix them soon.

I'm waiting for this to test this beautiful work.

**Reply by Alex-G** [reply]
> > There are some new compilation errors in the final version of the engine, but I'll fix them soon.
> 
> I'm waiting for this to test this beautiful work.

It is already fixed.
you can download let latest version of the plugin here: https://github.com/Sixze/ALS-Refactored/archive/refs/heads/main.zip
